src/config/appSettings.ts

```diff
@@ -9,15 +9,15 @@ const appSettings = {
   rpcAccessToken: process.env.RPC_ACCESS_TOKEN,
   logLevel: process.env.LOG_LEVEL || 'debug',
   pollingInterval: Number(process.env.POLLING_INTERVAL) ?? 5000,
+  chunkSize: Number(process.env.CHUNK_SIZE) ?? 1000,
```

Comment:
Something I noticed now, starting from a clean state.
If I don't specify a `POLLING_INTERVAL` (or similar) `POLLING_INTERVAL` will be `undefined` and `Number(underfined)` is `NaN` resulting in a polling failure. Using `||` instead of `??` works. Same for other `??` props.